aspects/aspects.go

```diff
@@ -289,6 +279,18 @@ func parseRule(parent *aspectRule, ruleRaw interface{}) ([]*aspectRule, error) {
 		return nil, errors.New(`"storage" must be a string`)
 	}
 
+	requestRaw, ok := ruleMap["request"]
+	if !ok {
+		requestRaw = storage
```

Comment:
I noticed this behavior of defaulting to the storage is documented in the spec, it would be nice to have a comment here explaining the reasoning here.

Reply:
Added a comment, thanks